[2022-10-09T02:54:46.935+0000] {processor.py:156} INFO - Started process (PID=10478) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:54:46.942+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:54:46.945+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:54:46.944+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:54:48.659+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:54:48.691+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:54:48.683+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:54:48.694+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:54:48.791+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.897 seconds
[2022-10-09T02:55:22.381+0000] {processor.py:156} INFO - Started process (PID=10694) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:55:22.382+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:55:22.384+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:55:22.383+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:55:22.763+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:55:22.771+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:55:22.767+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:55:22.773+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:55:23.249+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.872 seconds
[2022-10-09T02:55:54.979+0000] {processor.py:156} INFO - Started process (PID=10914) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:55:54.980+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:55:54.981+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:55:54.981+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:55:55.369+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:55:55.377+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:55:55.373+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:55:55.379+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:55:55.394+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.418 seconds
[2022-10-09T02:56:29.685+0000] {processor.py:156} INFO - Started process (PID=11120) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:56:29.688+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:56:29.689+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:56:29.689+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:56:30.187+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:56:30.194+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:56:30.191+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:56:30.195+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:56:30.216+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.546 seconds
[2022-10-09T02:57:00.355+0000] {processor.py:156} INFO - Started process (PID=11333) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:57:00.355+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:57:00.356+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:57:00.356+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:57:00.743+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:57:00.753+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:57:00.749+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:57:00.755+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:57:00.778+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.426 seconds
[2022-10-09T02:57:31.149+0000] {processor.py:156} INFO - Started process (PID=11543) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:57:31.150+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:57:31.150+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:57:31.150+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:57:31.563+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:57:31.572+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:57:31.569+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:57:31.573+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:57:32.252+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.105 seconds
[2022-10-09T02:58:02.583+0000] {processor.py:156} INFO - Started process (PID=11762) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:58:02.584+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:58:02.585+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:58:02.584+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:58:02.938+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:58:02.946+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:58:02.942+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:58:02.948+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:58:02.976+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.396 seconds
[2022-10-09T02:58:35.971+0000] {processor.py:156} INFO - Started process (PID=11975) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:58:35.972+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:58:35.973+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:58:35.973+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:58:36.388+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:58:36.395+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:58:36.392+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:58:36.397+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:58:36.423+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.454 seconds
[2022-10-09T02:59:07.896+0000] {processor.py:156} INFO - Started process (PID=12185) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:59:07.897+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:59:07.898+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:59:07.898+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:59:08.275+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:59:08.284+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:59:08.280+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:59:08.285+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:59:08.312+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.420 seconds
[2022-10-09T02:59:38.858+0000] {processor.py:156} INFO - Started process (PID=12403) to work on /opt/airflow/dags/testDag.py
[2022-10-09T02:59:38.858+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T02:59:38.859+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:59:38.859+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T02:59:39.238+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T02:59:39.245+0000] {logging_mixin.py:117} INFO - [2022-10-09T02:59:39.241+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T02:59:39.246+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T02:59:39.263+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.409 seconds
[2022-10-09T03:00:10.868+0000] {processor.py:156} INFO - Started process (PID=12613) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:00:10.869+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:00:10.870+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:00:10.870+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:00:11.300+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:00:11.309+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:00:11.305+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:00:11.311+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:00:11.332+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.467 seconds
[2022-10-09T03:00:46.451+0000] {processor.py:156} INFO - Started process (PID=12823) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:00:46.452+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:00:46.454+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:00:46.453+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:00:46.885+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:00:46.892+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:00:46.889+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:00:46.893+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:00:46.921+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.474 seconds
[2022-10-09T03:01:18.461+0000] {processor.py:156} INFO - Started process (PID=13042) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:01:18.462+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:01:18.462+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:01:18.462+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:01:18.891+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:01:18.900+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:01:18.896+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:01:18.901+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:01:18.930+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.474 seconds
[2022-10-09T03:01:49.088+0000] {processor.py:156} INFO - Started process (PID=13259) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:01:49.089+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:01:49.090+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:01:49.090+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:01:49.504+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:01:49.513+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:01:49.509+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:01:49.515+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:01:49.536+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.453 seconds
[2022-10-09T03:02:19.626+0000] {processor.py:156} INFO - Started process (PID=13473) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:02:19.627+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:02:19.628+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:02:19.628+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:02:20.088+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:02:20.097+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:02:20.093+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:02:20.098+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:02:20.118+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.494 seconds
[2022-10-09T03:02:52.571+0000] {processor.py:156} INFO - Started process (PID=13692) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:02:52.572+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:02:52.574+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:02:52.574+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:02:53.079+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:02:53.090+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:02:53.085+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:02:53.091+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:02:53.115+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.548 seconds
[2022-10-09T03:03:23.399+0000] {processor.py:156} INFO - Started process (PID=13882) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:03:23.400+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:03:23.401+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:03:23.401+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:03:23.772+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:03:23.781+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:03:23.777+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:03:23.782+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:03:23.804+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.409 seconds
[2022-10-09T03:03:54.642+0000] {processor.py:156} INFO - Started process (PID=14092) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:03:54.643+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:03:54.644+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:03:54.644+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:03:55.038+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:03:55.044+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:03:55.041+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:03:55.045+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:03:55.062+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.425 seconds
[2022-10-09T03:04:30.795+0000] {processor.py:156} INFO - Started process (PID=14306) to work on /opt/airflow/dags/testDag.py
[2022-10-09T03:04:30.796+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T03:04:30.798+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:04:30.797+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T03:04:31.203+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T03:04:31.210+0000] {logging_mixin.py:117} INFO - [2022-10-09T03:04:31.207+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T03:04:31.212+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T03:04:31.237+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.444 seconds
[2022-10-09T12:31:03.489+0000] {processor.py:156} INFO - Started process (PID=14500) to work on /opt/airflow/dags/testDag.py
[2022-10-09T12:31:03.492+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:14:09.169+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:14:09.168+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:14:19.286+0000] {processor.py:156} INFO - Started process (PID=14701) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:14:19.287+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:14:19.287+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:14:19.287+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:14:19.685+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:14:19.693+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:14:19.688+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:14:19.695+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:14:20.404+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.121 seconds
[2022-10-09T17:14:51.787+0000] {processor.py:156} INFO - Started process (PID=16965) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:14:51.788+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:14:51.789+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:14:51.789+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:14:52.347+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:14:52.365+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:14:52.360+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:14:52.366+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:14:52.770+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.991 seconds
[2022-10-09T17:15:24.516+0000] {processor.py:156} INFO - Started process (PID=17175) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:15:24.517+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:15:24.518+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:15:24.518+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:15:24.968+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:15:24.976+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:15:24.973+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:15:24.977+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:15:24.996+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.483 seconds
[2022-10-09T17:15:55.208+0000] {processor.py:156} INFO - Started process (PID=17394) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:15:55.210+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:15:55.211+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:15:55.210+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:15:55.603+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:15:55.611+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:15:55.607+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:15:55.612+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:15:55.646+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.441 seconds
[2022-10-09T17:16:26.147+0000] {processor.py:156} INFO - Started process (PID=17592) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:16:26.147+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:16:26.148+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:16:26.148+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:16:26.568+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:16:26.577+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:16:26.573+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:16:26.578+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:16:26.597+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.453 seconds
[2022-10-09T17:16:57.688+0000] {processor.py:156} INFO - Started process (PID=17721) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:16:57.689+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:16:57.689+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:16:57.689+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:16:58.076+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:16:58.085+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:16:58.081+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:16:58.086+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:16:58.103+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.418 seconds
[2022-10-09T17:17:28.132+0000] {processor.py:156} INFO - Started process (PID=17941) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:17:28.133+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:17:28.134+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:17:28.134+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:17:28.553+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:17:28.562+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:17:28.558+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:17:28.564+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:17:28.587+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.457 seconds
[2022-10-09T17:17:58.731+0000] {processor.py:156} INFO - Started process (PID=18153) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:17:58.731+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:17:58.732+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:17:58.732+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:17:59.148+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:17:59.156+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:17:59.152+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:17:59.157+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:17:59.177+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.449 seconds
[2022-10-09T17:18:29.432+0000] {processor.py:156} INFO - Started process (PID=18335) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:18:29.433+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:18:29.434+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:18:29.434+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:18:31.645+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:18:31.656+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:18:31.652+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:18:31.657+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:18:31.681+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 2.252 seconds
[2022-10-09T17:19:01.998+0000] {processor.py:156} INFO - Started process (PID=18545) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:19:02.000+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:19:02.001+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:19:02.001+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:19:02.415+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:19:02.423+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:19:02.419+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:19:02.424+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:19:02.975+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.981 seconds
[2022-10-09T17:19:33.554+0000] {processor.py:156} INFO - Started process (PID=18755) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:19:33.555+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:19:33.556+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:19:33.556+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:19:34.031+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:19:34.042+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:19:34.035+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:19:34.043+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:19:34.070+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.519 seconds
[2022-10-09T17:20:04.111+0000] {processor.py:156} INFO - Started process (PID=18919) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:20:04.112+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:20:04.113+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:20:04.113+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:20:04.507+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:20:04.514+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:20:04.511+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:20:04.515+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:20:04.540+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.432 seconds
[2022-10-09T17:20:35.137+0000] {processor.py:156} INFO - Started process (PID=19129) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:20:35.138+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:20:35.139+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:20:35.139+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:20:35.552+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:20:35.559+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:20:35.555+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:20:35.560+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:20:35.587+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.452 seconds
[2022-10-09T17:21:05.722+0000] {processor.py:156} INFO - Started process (PID=19339) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:21:05.724+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:21:05.725+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:21:05.725+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:21:09.075+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:21:09.083+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:21:09.079+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:21:09.085+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:21:09.104+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 3.391 seconds
[2022-10-09T17:21:39.411+0000] {processor.py:156} INFO - Started process (PID=19558) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:21:39.411+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:21:39.412+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:21:39.412+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:21:39.845+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:21:39.854+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:21:39.850+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:21:39.855+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:21:39.873+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.465 seconds
[2022-10-09T17:22:10.903+0000] {processor.py:156} INFO - Started process (PID=19768) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:22:10.904+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:22:10.905+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:22:10.905+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:22:11.306+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:22:11.314+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:22:11.310+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:22:11.316+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:22:11.337+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.437 seconds
[2022-10-09T17:22:41.535+0000] {processor.py:156} INFO - Started process (PID=19977) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:22:41.536+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:22:41.537+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:22:41.537+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:22:41.998+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:22:42.006+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:22:42.002+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:22:42.007+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:22:42.483+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.951 seconds
[2022-10-09T17:23:14.106+0000] {processor.py:156} INFO - Started process (PID=20199) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:23:14.107+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:23:14.108+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:23:14.108+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:23:14.519+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:23:14.526+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:23:14.523+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:23:14.528+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:23:14.554+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.451 seconds
[2022-10-09T17:23:44.749+0000] {processor.py:156} INFO - Started process (PID=20409) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:23:44.750+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:23:44.751+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:23:44.751+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:23:45.151+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:23:45.159+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:23:45.155+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:23:45.161+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:23:45.181+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.434 seconds
[2022-10-09T17:24:15.352+0000] {processor.py:156} INFO - Started process (PID=20618) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:24:15.354+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:24:15.355+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:24:15.354+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:24:15.774+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:24:15.781+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:24:15.778+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:24:15.783+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:24:15.803+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.454 seconds
[2022-10-09T17:24:45.967+0000] {processor.py:156} INFO - Started process (PID=20828) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:24:45.968+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:24:45.969+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:24:45.969+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:24:46.399+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:24:46.405+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:24:46.403+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:24:46.406+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:24:46.428+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.466 seconds
[2022-10-09T17:25:16.772+0000] {processor.py:156} INFO - Started process (PID=21045) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:25:16.773+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:25:16.774+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:25:16.773+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:25:17.236+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:25:17.244+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:25:17.240+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:25:17.246+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:25:17.268+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.502 seconds
[2022-10-09T17:25:47.530+0000] {processor.py:156} INFO - Started process (PID=21257) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:25:47.531+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:25:47.531+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:25:47.531+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:25:47.980+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:25:47.987+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:25:47.983+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:25:47.989+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:25:48.013+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.487 seconds
[2022-10-09T17:26:19.010+0000] {processor.py:156} INFO - Started process (PID=21467) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:26:19.010+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:26:19.011+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:26:19.011+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:26:19.412+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:26:19.419+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:26:19.416+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:26:19.421+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:26:19.438+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.431 seconds
[2022-10-09T17:26:49.707+0000] {processor.py:156} INFO - Started process (PID=21677) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:26:49.710+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:26:49.711+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:26:49.711+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:26:50.126+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:26:50.135+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:26:50.131+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:26:50.136+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:26:50.154+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.450 seconds
[2022-10-09T17:27:20.371+0000] {processor.py:156} INFO - Started process (PID=21894) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:27:20.372+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:27:20.373+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:27:20.373+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:27:20.795+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:27:20.802+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:27:20.798+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:27:20.803+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:27:20.831+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.463 seconds
[2022-10-09T17:27:50.929+0000] {processor.py:156} INFO - Started process (PID=22093) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:27:50.930+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:27:50.931+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:27:50.931+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:27:51.354+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:27:51.362+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:27:51.359+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:27:51.363+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:27:51.383+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.457 seconds
[2022-10-09T17:28:21.797+0000] {processor.py:156} INFO - Started process (PID=22303) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:28:21.798+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:28:21.800+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:28:21.800+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:28:22.212+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:28:22.220+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:28:22.216+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:28:22.221+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:28:22.243+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.448 seconds
[2022-10-09T17:28:53.785+0000] {processor.py:156} INFO - Started process (PID=22520) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:28:53.786+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:28:53.788+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:28:53.788+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:28:54.272+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:28:54.281+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:28:54.277+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:28:54.282+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:28:54.303+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.522 seconds
[2022-10-09T17:29:24.491+0000] {processor.py:156} INFO - Started process (PID=22733) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:29:24.492+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:29:24.494+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:29:24.494+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:29:24.906+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:29:24.913+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:29:24.910+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:29:24.915+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:29:24.940+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.451 seconds
[2022-10-09T17:29:56.742+0000] {processor.py:156} INFO - Started process (PID=22940) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:29:56.743+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:29:56.744+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:29:56.744+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:29:57.178+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:29:57.184+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:29:57.181+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:29:57.185+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:29:57.205+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.467 seconds
[2022-10-09T17:30:27.794+0000] {processor.py:156} INFO - Started process (PID=23139) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:30:27.795+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:30:27.795+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:30:27.795+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:30:28.160+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:30:28.168+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:30:28.164+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:30:28.169+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:30:28.895+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.104 seconds
[2022-10-09T17:30:58.975+0000] {processor.py:156} INFO - Started process (PID=23358) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:30:58.976+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:30:58.977+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:30:58.976+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:30:59.434+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:30:59.440+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:30:59.437+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:30:59.442+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:30:59.468+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.496 seconds
[2022-10-09T17:31:29.969+0000] {processor.py:156} INFO - Started process (PID=23568) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:31:29.970+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:31:29.971+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:31:29.971+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:31:30.326+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:31:30.333+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:31:30.330+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:31:30.334+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:31:30.358+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.392 seconds
[2022-10-09T17:32:01.266+0000] {processor.py:156} INFO - Started process (PID=23778) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:32:01.272+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:32:01.274+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:32:01.273+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:32:01.746+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:32:01.754+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:32:01.750+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:32:01.756+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:32:01.776+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.513 seconds
[2022-10-09T17:32:32.471+0000] {processor.py:156} INFO - Started process (PID=23988) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:32:32.472+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:32:32.473+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:32:32.473+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:32:32.853+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:32:32.862+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:32:32.858+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:32:32.863+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:32:32.889+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.422 seconds
[2022-10-09T17:33:03.667+0000] {processor.py:156} INFO - Started process (PID=24199) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:33:03.668+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:33:03.669+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:33:03.669+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:33:04.077+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:33:04.086+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:33:04.082+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:33:04.087+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:33:04.106+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.442 seconds
[2022-10-09T17:33:34.920+0000] {processor.py:156} INFO - Started process (PID=24415) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:33:34.921+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:33:34.922+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:33:34.922+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:33:35.324+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:33:35.332+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:33:35.329+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:33:35.333+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:33:35.359+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.442 seconds
[2022-10-09T17:34:06.147+0000] {processor.py:156} INFO - Started process (PID=24627) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:34:06.148+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:34:06.150+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:34:06.149+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:34:06.602+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:34:06.610+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:34:06.606+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:34:06.611+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:34:06.639+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.497 seconds
[2022-10-09T17:34:36.883+0000] {processor.py:156} INFO - Started process (PID=24828) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:34:36.884+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:34:36.885+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:34:36.885+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:34:37.296+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:34:37.305+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:34:37.302+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:34:37.306+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:34:37.331+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.451 seconds
[2022-10-09T17:35:08.169+0000] {processor.py:156} INFO - Started process (PID=25049) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:35:08.170+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:35:08.171+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:35:08.171+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:35:08.551+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:35:08.558+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:35:08.554+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:35:08.559+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:35:08.577+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.411 seconds
[2022-10-09T17:35:39.360+0000] {processor.py:156} INFO - Started process (PID=25259) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:35:39.361+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:35:39.362+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:35:39.361+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:35:39.714+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:35:39.728+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:35:39.724+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:35:39.729+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:35:39.762+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.406 seconds
[2022-10-09T17:36:10.830+0000] {processor.py:156} INFO - Started process (PID=25468) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:36:10.830+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:36:10.831+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:36:10.831+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:36:12.074+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:36:12.080+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:36:12.077+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:36:12.081+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:36:12.097+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.270 seconds
[2022-10-09T17:36:42.383+0000] {processor.py:156} INFO - Started process (PID=25678) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:36:42.384+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:36:42.385+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:36:42.385+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:36:42.838+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:36:42.847+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:36:42.842+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:36:42.849+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:36:42.881+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.503 seconds
[2022-10-09T17:37:13.084+0000] {processor.py:156} INFO - Started process (PID=25901) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:37:13.085+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:37:13.086+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:37:13.086+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:37:13.589+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:37:13.598+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:37:13.594+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:37:13.599+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:37:13.621+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.542 seconds
[2022-10-09T17:37:43.699+0000] {processor.py:156} INFO - Started process (PID=26121) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:37:43.700+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:37:43.700+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:37:43.700+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:37:44.129+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:37:44.138+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:37:44.135+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:37:44.140+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:37:44.165+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.468 seconds
[2022-10-09T17:38:14.319+0000] {processor.py:156} INFO - Started process (PID=26331) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:38:14.319+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:38:14.320+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:38:14.320+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:38:14.754+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:38:14.762+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:38:14.758+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:38:14.764+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:38:15.773+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.457 seconds
[2022-10-09T17:38:46.583+0000] {processor.py:156} INFO - Started process (PID=26524) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:38:46.584+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:38:46.584+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:38:46.584+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:38:47.022+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:38:47.031+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:38:47.027+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:38:47.033+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:38:47.056+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.476 seconds
[2022-10-09T17:39:18.355+0000] {processor.py:156} INFO - Started process (PID=26738) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:39:18.356+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:39:18.357+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:39:18.356+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:39:18.788+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:39:18.797+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:39:18.793+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:39:18.799+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:39:18.819+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.466 seconds
[2022-10-09T17:39:49.007+0000] {processor.py:156} INFO - Started process (PID=26923) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:39:49.008+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:39:49.010+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:39:49.010+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:39:49.582+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:39:49.598+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:39:49.589+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:39:49.601+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:39:50.017+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.013 seconds
[2022-10-09T17:40:20.150+0000] {processor.py:156} INFO - Started process (PID=27135) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:40:20.150+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:40:20.151+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:40:20.151+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:40:20.586+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:40:20.594+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:40:20.591+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:40:20.595+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:40:20.614+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.468 seconds
[2022-10-09T17:40:50.734+0000] {processor.py:156} INFO - Started process (PID=27345) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:40:50.735+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:40:50.736+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:40:50.736+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:40:51.355+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:40:51.363+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:40:51.359+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:40:51.364+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:40:51.382+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.651 seconds
[2022-10-09T17:41:22.044+0000] {processor.py:156} INFO - Started process (PID=27555) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:41:22.045+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:41:22.047+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:41:22.047+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:41:22.624+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:41:22.634+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:41:22.630+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:41:22.636+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:41:22.660+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.618 seconds
[2022-10-09T17:41:53.466+0000] {processor.py:156} INFO - Started process (PID=27765) to work on /opt/airflow/dags/testDag.py
[2022-10-09T17:41:53.468+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T17:41:53.469+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:41:53.468+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T17:41:54.007+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T17:41:54.015+0000] {logging_mixin.py:117} INFO - [2022-10-09T17:41:54.011+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T17:41:54.017+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T17:41:54.036+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.574 seconds
[2022-10-09T22:37:43.407+0000] {processor.py:156} INFO - Started process (PID=27923) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:37:43.409+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:37:43.411+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:37:43.410+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:37:44.575+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:37:44.593+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:37:44.585+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:37:44.596+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:37:44.653+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.254 seconds
[2022-10-09T22:38:15.277+0000] {processor.py:156} INFO - Started process (PID=28128) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:38:15.278+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:38:15.279+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:38:15.279+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:38:15.639+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:38:15.646+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:38:15.643+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:38:15.647+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:38:15.666+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.391 seconds
[2022-10-09T22:38:46.367+0000] {processor.py:156} INFO - Started process (PID=28338) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:38:46.369+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:38:46.370+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:38:46.370+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:38:47.025+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:38:47.036+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:38:47.031+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:38:47.038+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:38:47.064+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.701 seconds
[2022-10-09T22:39:17.409+0000] {processor.py:156} INFO - Started process (PID=28548) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:39:17.411+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:39:17.413+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:39:17.412+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:39:18.142+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:39:18.150+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:39:18.146+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:39:18.151+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:39:18.171+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.768 seconds
[2022-10-09T22:39:48.535+0000] {processor.py:156} INFO - Started process (PID=28758) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:39:48.535+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:39:48.536+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:39:48.536+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:39:48.922+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:39:48.929+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:39:48.926+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:39:48.930+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:39:49.603+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 1.073 seconds
[2022-10-09T22:40:19.733+0000] {processor.py:156} INFO - Started process (PID=28967) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:40:19.734+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:40:19.735+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:40:19.735+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:40:20.160+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:40:20.170+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:40:20.165+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:40:20.172+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:40:20.193+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.462 seconds
[2022-10-09T22:40:50.420+0000] {processor.py:156} INFO - Started process (PID=29176) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:40:50.420+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:40:50.421+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:40:50.421+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:40:50.849+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:40:50.857+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:40:50.853+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:40:50.859+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:40:50.886+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.468 seconds
[2022-10-09T22:41:21.248+0000] {processor.py:156} INFO - Started process (PID=29386) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:41:21.249+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:41:21.250+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:41:21.250+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:41:21.632+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:41:21.640+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:41:21.637+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:41:21.642+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:41:21.659+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.414 seconds
[2022-10-09T22:41:52.537+0000] {processor.py:156} INFO - Started process (PID=29605) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:41:52.539+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:41:52.540+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:41:52.540+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:41:53.038+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:41:53.045+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:41:53.042+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:41:53.046+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:41:53.071+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.537 seconds
[2022-10-09T22:42:23.318+0000] {processor.py:156} INFO - Started process (PID=29814) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:42:23.319+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:42:23.319+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:42:23.319+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:42:23.768+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:42:23.776+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:42:23.773+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:42:23.777+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:42:23.798+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.483 seconds
[2022-10-09T22:42:53.936+0000] {processor.py:156} INFO - Started process (PID=30026) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:42:53.937+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:42:53.938+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:42:53.937+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:42:54.375+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:42:54.382+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:42:54.379+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:42:54.383+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:42:54.400+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.466 seconds
[2022-10-09T22:43:24.469+0000] {processor.py:156} INFO - Started process (PID=30238) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:43:24.470+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:43:24.471+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:43:24.471+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:43:25.324+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:43:25.331+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:43:25.328+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:43:25.332+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:43:25.351+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.885 seconds
[2022-10-09T22:43:55.518+0000] {processor.py:156} INFO - Started process (PID=30473) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:43:55.519+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:43:55.519+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:43:55.519+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:43:55.902+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:43:55.910+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:43:55.906+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:43:55.911+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:43:55.936+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.422 seconds
[2022-10-09T22:44:28.278+0000] {processor.py:156} INFO - Started process (PID=30650) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:44:28.279+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:44:28.280+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:44:28.280+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:44:28.672+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:44:28.679+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:44:28.675+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:44:28.680+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:44:28.705+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.429 seconds
[2022-10-09T22:44:59.326+0000] {processor.py:156} INFO - Started process (PID=30869) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:44:59.326+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:44:59.327+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:44:59.327+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:44:59.703+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:44:59.711+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:44:59.708+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:44:59.712+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:44:59.742+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.419 seconds
[2022-10-09T22:45:29.841+0000] {processor.py:156} INFO - Started process (PID=31078) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:45:29.841+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:45:29.843+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:45:29.842+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:45:30.269+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:45:30.276+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:45:30.273+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:45:30.277+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:45:30.294+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.457 seconds
[2022-10-09T22:46:00.580+0000] {processor.py:156} INFO - Started process (PID=31291) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:46:00.581+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:46:00.582+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:46:00.581+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:46:01.156+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:46:01.162+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:46:01.159+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:46:01.164+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:46:01.188+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.614 seconds
[2022-10-09T22:46:31.275+0000] {processor.py:156} INFO - Started process (PID=31516) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:46:31.276+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:46:31.277+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:46:31.277+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:46:31.681+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:46:31.690+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:46:31.686+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:46:31.691+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:46:31.716+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.444 seconds
[2022-10-09T22:47:04.826+0000] {processor.py:156} INFO - Started process (PID=31669) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:47:04.827+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:47:04.828+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:47:04.828+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:47:05.235+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:47:05.244+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:47:05.239+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:47:05.245+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:47:05.265+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.441 seconds
[2022-10-09T22:47:35.528+0000] {processor.py:156} INFO - Started process (PID=31862) to work on /opt/airflow/dags/testDag.py
[2022-10-09T22:47:35.529+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/testDag.py for tasks to queue
[2022-10-09T22:47:35.530+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:47:35.530+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/testDag.py
[2022-10-09T22:47:36.323+0000] {logging_mixin.py:117} INFO - Error No module named 'scraper'
[2022-10-09T22:47:36.338+0000] {logging_mixin.py:117} INFO - [2022-10-09T22:47:36.332+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/testDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/testDag.py", line 42, in <module>
    python_callable=getMakeYrPrice
NameError: name 'getMakeYrPrice' is not defined
[2022-10-09T22:47:36.340+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/testDag.py
[2022-10-09T22:47:36.370+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/testDag.py took 0.848 seconds
